### ISSUE TITLE
Please support feature and release branch model

### ISSUE THREAD

**Issue by skurlow** [SmartBear/git-en-boite#343]
We don't use master and develop branches so that we can have 3 releases being developed concurrently.

1) We have a stream of work focusing on planned fixing of bugs in production
2) We have another stream of work focusing on a new epic
3) We have urgent work focusing on a hotfix of a breaking feature in production

Before the above start the release in prod could be v1.2.3

We plan to designate work of 1) to go into 1.3.0
We plan to designate work of 2) to go into 2.0.0
We plan to designate work of 3) to go into 1.2.4

Work for 1) means a release branch called release/1.3.0 is created from release/1.2.3 and all feature branches off the new release/1.3.0 will be merged back into release/1.3.0

Work for 2) means a release branch called release/2.0.0 is created from release/1..2.3 and all feature branches off the new release/2.0.0 will be merged back into release/2.0.0

Work for 3) means a release branch called release/1.2.4 is created from release/1.2.3 and all feature branches off the new release/1.2.4 will be merged back into release/1.2.4

When work for 3) is complete. It is what is planned to complete first. Then once complete we merge release/1.2.4 into release/1.3.0. Then we merge release/1.3.0 into release/2.0.0. Finally release/1.2.4 is tagged and release branch deleted.

When work for 1) is complete. It is what is planned to complete second. Then once complete we merge release/1.3.0 into release/2.0.0. Finally release/1.3.0 is tagged and release branch deleted.

And so on for subsequent releases.

So bottom line is we don't use master and develop branches and we would appreciate if you support a feature and release branch model where we can be viewing/editing features in feature branches and viewing them in release branches. Our release branches are protected so editing of feature files would not be allowed there.

**Comment by mattwynne** [comment]
Hi @skurlow thanks for your feedback.

This issue tracker is really just for git-en-boite, which is part of the mechanism for delivering the Ghekin Editor in CucumberStudio, but not the best place to give us feedback on the editor itself.

There are two places I'd recommend for discussing CucumberStudio with the team:

1) Inside the [CucumberStudio app](https://studio.cucumber.io/) itself, use the Intercom widget to chat with the team. @cbliard our product owner is in there and you can chat with him there.

2) The [SmartBear Community](https://community.smartbear.com/t5/CucumberStudio/bd-p/CucumberStudio) forum 

I *think* the workflow you're talking about should be supported by CucumberStudio, so if you haven't already, please give it a try!
